src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -58,35 +65,62 @@
     disabled: true,
   }) %}
 {% endif %}
+
 {% if _invalid %}
   {% set _input = _input|merge({
     invalid: true,
   }) %}
+  {% if _id %}
+    {% set _aria_describedby = _aria_describedby ? _aria_describedby ~ ' ' ~ _id ~ '-invalid' : _id ~ '-invalid' %}
+  {% endif %}
+  {% set _input_extra_attributes = _input_extra_attributes|merge([
+    { name: 'aria-invalid', value: 'true' }
+  ]) %}
 {% endif %}
+
 {% if _required %}
   {% set _input = _input|merge({
     required: true,
   }) %}
 {% endif %}
 
 {% if _input.input_type == 'range' %}
-  {% set extra_attributes = [
+  {% set extra_attributes = extra_attributes|default([])|merge([
     { name: 'data-ecl-range', value: true },
     { name: 'data-ecl-auto-init', value: 'Range' },
-  ] %}
-{% endif %}
-
-{% if _input.input_type == 'rating-field' and _helper_id is not empty %}
- {% set extra_attributes = [
-  { name: 'aria-describedby', value: _helper_id }
- ] %}
+  ]) %}
 {% endif %}
 
 {% if _input.input_type == 'file' %}
-  {% set extra_attributes = [
+  {% set extra_attributes = extra_attributes|default([])|merge([
     { name: 'data-ecl-file-upload-group', value: true },
     { name: 'data-ecl-autoinit', value: 'FileUpload' },
-  ] %}
+  ]) %}
+{% endif %}
+
+{% if _helper_text is not empty %}
```

Comment:
i think this is not enough, we risk to render an empty aria-describedby like this, we should also check for the _aria_describedby variable for its emptyness.